src/lib/atoms/performance/loadable.ts

```diff
@@ -55,3 +55,20 @@ export const errorsAtom = atom(get => {
 
   return errors
 })
+
+// Jotai Devtools debug labels
+// Note: local atoms are labeled for clarity even if not exported
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
@CharlieHelps - why the `@ts-ignore` here? is this not something we can setup in the tsconfig?